ts/DataGrid/.eslintrc

```diff
@@ -1,10 +1,13 @@
 {
     "extends": "../.eslintrc",
     "rules": {
-      "jsdoc/require-param": 0,
-      "jsdoc/require-param-description": 0,
+      "jsdoc/require-jsdoc": 2,
```

Comment:
Do we need it? If so, let's keep it inline with what we have for Dashboards.

Reply:
It just helps with catching mistakes in doclets. Can be changed also in Dashboards, but I would do it in a different task when we'll be working on the new API docs.